dao/mongo.go

```diff
@@ -67,15 +67,15 @@ type MongoService struct {
 func (m *MongoService) SaveE5Error(companyNumber, reference string, action e5.Action) error {
 	dao, err := m.GetPayableResource(companyNumber, reference)
 	if err != nil {
-		log.Error(err, log.Data{"company_number": companyNumber, "lfp_reference": reference})
+		log.Error(err, log.Data{"company_number": companyNumber, "penalty_reference": reference})
 		return err
 	}
 
 	filter := bson.M{"_id": dao.ID}
 	update := bson.D{
 		{
-			"$set", bson.D{
-				{"e5_command_error", string(action)},
+			Key: "$set", Value: bson.D{
```

Comment:
What's the reason behind this code style - is this something suggested by SonarQube?
I'm not familiar with Go :)

Reply:
Yeah, it was flagged in the IDE and I updated based on the suggestion.